playground/integration_test/playground_flow_test.dart

```diff
@@ -0,0 +1,143 @@
+import 'package:demo_ui_components/demo_ui_components.dart';
+import 'package:flutter/material.dart';
+import 'package:patrol/patrol.dart';
+import 'package:playground/main.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
+  patrolTest(
+    'Page with forced max height and background color flow',
+    ($) async {
+      await $.pumpWidgetAndSettle(
+        const MaterialApp(
+          home: DemoApp(
+            androidSdkVersion: 30,
+          ),
+        ),
+      );
+      //Page with forced max height and background color flow
+      expect($('Wolt Modal Sheet Playground'), findsOneWidget);
```

Comment:
Thanks Furkan for the PR! We aim to not use text as finders as they likely to change more often compared to other finders. Could you please update to use other finders like; keys, type, widget, etc.

Reply:
Yes, you are absolutely right, I made it a little bit like a demo, from now on I will only use types and keys.